blocks_vertical/default_toolbox.js

```diff
@@ -703,6 +740,11 @@ Blockly.Blocks.defaultToolbox = '<xml id="toolbox-categories" style="display: no
   '<category name="Extensions" id="extensions" colour="#FF6680" secondaryColour="#FF4D6A" ' +
     'iconURI="../media/extensions/wedo2-block-icon.svg" showStatusButton="true">' +
     '<block type="extension_pen_down" id="extension_pen_down"></block>' +
+    '<block type="extension_checkbox_test">' +
+      '<value name="CHECKBOX">' +
+        '<shadow type="checkbox"></shadow>' +
+      '</value>' +
+    '</block>' +
```

Comment:
Not exactly required, but let's put all NitroBolt specific extension tests at the top, and separate them visibly from the other blocks (top of the extension test section, and simple <sep> tag)